### ISSUE TITLE
Create a second level of script wrapping for cluster jobs

### ISSUE THREAD

**Issue by natacha-beck** [aces/cbrain#64]
In order to improve robustness on clusters that do not
provide reliable information about active jobs, Tarek
and I discussed and bit and came up with an improvement.

The user-supplied bash script is no longer to be submitted
as thejob, instead a wrapper bash script is submitted, which
would look like this:

```
got_sigterm() {
  touch terminated; exit 20
}

trap got_sigterm SIGTERM

touch started
bash the_job_script
touch finished
```

This way we can more or less detect cases where the cluster's scheduler
is lying to CBRAIN, and properly wait some more if we think the
jobs is invisibly on CPU, or if it has never been executed at all
(two common problems).

**Comment by prioux** [comment]
Related to issue #133 : both can be done at the same time.

**Comment by prioux** [comment]
Fixed by #524
